src/interfaces.ts

```diff
@@ -1014,17 +1014,30 @@ export interface ExtraSymbolData {
      * Was this a result of a callfunc?
      */
     isFromCallFunc?: boolean;
+    /**
+     * Some symbols can be used in the same node they are defined in
+     * For example, loop variables in for loops
+     */
+    canUseInDefinedNode?: boolean;
```

Comment:
What does `node` mean in this case? `AstNode`? Perhaps we could include the word `ast` in `canUseInDefinedAstNode` to help prevent confusion with scenegraph nodes (or some other abstract "node" type)